xarray/backends/api.py

```diff
@@ -1611,7 +1617,27 @@ def open_mfdataset(
         open_ = open_dataset
         getattr_ = getattr
 
-    datasets = [open_(p, **open_kwargs) for p in paths1d]
+    if errors in ("raise", "warn", "ignore"):
```

Comment:
```suggestion
    if errors not in ("raise", "warn", "ignore"):
        raise ValueError(f"'errors' must be 'raise', 'warn' or 'ignore', got '{errors}'")
```
And then de-dent everything below.